ci/src/cI_target.ml

```diff
@@ -87,6 +87,10 @@ let path = function
 
 type v = [ `PR of PR.t | `Ref of Ref.t ]
 
+let pp_v f = function
+  | `PR pr  -> Fmt.pf f "pr/%a" PR.pp pr
+  | `Ref x -> Fmt.pf f "refs/%a" Ref.pp x
```

Comment:
yes not sure where this is used. I've tried to not use `s` in the datakit/github bridge, but indeed Git is putting `s` everywhere.